cluster-config.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+nodes:
+  - role: control-plane
+  - role: worker
+    extraPortMappings:
+      - containerPort: 30080
```

Comment:
this block needs to map directly to https://github.com/zpratt/local-openfaas-workbench/pull/2/files#diff-f124141f00da4f799a89112575608d3cR41-R44